tests/go.mod

```diff
@@ -10,8 +10,8 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
 	helm.sh/helm/v3 v3.8.0
 	k8s.io/api v0.23.1
-	k8s.io/apimachinery v0.23.1
-	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
+	k8s.io/apimachinery v0.23.3
```

Comment:
dependency updates like this should ideally be in a separate PR, because it makes it easier to revert independently if needed

Reply:
Will revert this change for now.